README.md

```diff
@@ -1,15 +1,52 @@
 # ReTither
+## To contribute
+Fork the this repo then clone from forked repo from your github account 
+
+```$git clone https://github.com/your_username/ReTither.git```  
+
+or   
+```$git clone git@github.com:your_username/ReTither.git```
+
 
 To start your Phoenix server:
 
   * Install dependencies with `mix deps.get`
-  * Create and migrate your database with `mix ecto.setup`
-  * Install Node.js dependencies with `cd assets && npm install`
+  * Create and migrate your database with `mix ecto.create && mix ecto.migrate`
+  * Install Node.js dependencies with `cd assets && npm install` 
   * Start Phoenix endpoint with `mix phx.server`
 
 Now you can visit [`localhost:4000`](http://localhost:4000) from your browser.
 
-Ready to run in production? Please [check our deployment guides](https://hexdocs.pm/phoenix/deployment.html).
+create remote with name upstream i.e 
+
+```$git remote add upstream https://github.com/championer-org/ReTither.git```
+
+ or
+
+```$git remote add upstream git@github.com:championer-org/ReTither.git```
+
+Before working on issue ensure your local repo is upto date by running
+
+```$git pull upstream develop```
+
+When creating a branch ensure it has an issue number
+
+For example issue #26 create user should be have branch name
+
+`3-update-contribution-readme`
+
+Run 
+
+```mix check_lints``` and    ```mix test```  and solve any arising issues before pushing
+
+Upon making changes push to your remote 
+
+```$git push origin branchname```
```

Comment:
Let's encourage people to push to `upstream`

```$ git push upstream branchname```

Reply:
Let's not waste time on documentation, yet. Good job getting this done.

People might fork it, to get started right away, while they wait for us to give them collaborator access. Therefore the fork is ok.